conversion/conversion.go

```diff
@@ -582,7 +582,7 @@ func UpdateDDLForeignKeys(ctx context.Context, adminClient *database.DatabaseAdm
 		return nil
 	}
 	msg := fmt.Sprintf("Updating schema of database %s with foreign key constraints ...", dbURI)
```

Comment:
Should we add some sort of a warning message before updating foreign keys that says
"Foreign Key Updates can be slow (even if no data is loaded) due to limitations on spanner side" or something similar?

Reply:
I think that sounds reasonable, especially if number of foreign keys is > 100 or something. In that case, you can also mention about the MaxWorkers variable that customers can tweak.